packages/uops-dashboard/postcss.config.mjs

```diff
@@ -5,4 +5,5 @@ const config = {
   },
 }
 
+// biome-ignore lint/style/noDefaultExport: <explanation>
```

Comment:
<img width="472" alt="image" src="https://github.com/user-attachments/assets/15032b8e-f98a-4bdc-a9b3-1751f080876d">

Reply:
Fixed, was not referencing root biome config file